plugins/webp-uploads/hooks.php

```diff
@@ -681,38 +681,54 @@ function webp_uploads_update_featured_image( $html, $post_id, $attachment_id ) {
  */
 function webp_uploads_wepb_fallback() {
 	// Get mime type transforms for the site.
-	$transforms = webp_uploads_get_upload_image_mime_transforms();
-
-	// We need to add fallback only if jpeg alternatives for the webp images are enabled for the server.
-	$preserve_jpegs_for_jpeg_transforms = isset( $transforms['image/jpeg'] ) && in_array( 'image/jpeg', $transforms['image/jpeg'], true ) && in_array( 'image/webp', $transforms['image/jpeg'], true );
-	$preserve_jpegs_for_webp_transforms = isset( $transforms['image/webp'] ) && in_array( 'image/jpeg', $transforms['image/webp'], true ) && in_array( 'image/webp', $transforms['image/webp'], true );
-	if ( ! $preserve_jpegs_for_jpeg_transforms && ! $preserve_jpegs_for_webp_transforms ) {
+	$transforms   = webp_uploads_get_upload_image_mime_transforms();
+	$image_format = get_option( 'perflab_modern_image_format' );
+	$image_format = empty( $image_format ) ? 'webp' : $image_format;
```

Comment:
This would seem a bit safer since someone could populate the option with anything:

```suggestion
	$image_format = in_array( $image_format, array( 'webp', 'avif' ), true ) ? $image_format : 'webp';
```